### ISSUE TITLE
Marker armour stands in blocks do not display lighting correctly

### ISSUE THREAD

**Issue by dvitski** [jellysquid3/Sodium#68]
### Expected Behavior
![2020-07-08_20 52 52](https://user-images.githubusercontent.com/36608902/86964206-9b4f3480-c15d-11ea-8b46-17965e55fb94.png)

### Actual Behavior
![2020-07-08_20 45 28](https://user-images.githubusercontent.com/36608902/86964214-a2764280-c15d-11ea-90f1-5a3495f2a793.png)

### Reproduction Steps
```mcfunction
setblock ~ ~ ~ stone
summon armor_stand ~ ~ ~ {Marker:1,Fire:400}
```

### Related
#60 - Armour stands with the Marker tag do not have lighting

### System Information
- Java Version: 1.8.0_51 64bit
- CPU: 2x Intel(R) Celeron(R) N4000 CPU @ 1.10GHz
- GPU: Intel(R) UHD Graphics 600

**Comment by dvitski** [comment]
Seems to be fixed 👍

**Comment by dvitski** [comment]
This issue is back :) 

[Without Sodium](https://i.imgur.com/5KKrGxa.mp4)  
[With Sodium](https://i.imgur.com/XmPXv4q.mp4)

*Also, rather fittingly, the without sodium video is laggier* :+1:

**Comment by Darwin1546** [comment]
I confirm this issue. Armor stand lighting is incorrect

**Comment by dvitski** [comment]
Cleaning up my issues, this seems fixed in 1.19 (v0.4.2+build.16)
